client/src/panels/people/employee_prov.js

```diff
@@ -293,16 +129,22 @@ const calculate_common = (data) => {
     .domain([0,max])
     .range([0.2,1]);
 
-  return { years_by_province, color_scale };
+  return {
+    years_by_province,
+    color_scale,
+    context_years,
+    formatter,
+    includeNcr,
+  };
 };
 const calculate_funcs_by_level = {
   gov: function(){
     const { orgEmployeeRegion } = this.tables;
-    return calculate_common( people_years.map( year => orgEmployeeRegion.prov_code(year, false) ) );
+    return calculate_common( context_years.map( year => orgEmployeeRegion.prov_code(year, false) ) );
   },
   dept: function(subject){
     const { orgEmployeeRegion } = this.tables;
-    return calculate_common( people_years.map( year => orgEmployeeRegion.prov_code(year, subject.unique_id) ) );
+    return calculate_common( context_years.map( year => orgEmployeeRegion.prov_code(year, subject.unique_id) ) );
   },
 };
```

Comment:
Love how much you cleaned this whole file up!

Reply:
That wasn't a comment to do with that specific line, I just left it somewhere near the bottom of the file,